terraform/run.tf

```diff
@@ -0,0 +1,334 @@
+          }
+        }
+      }
+
+      env {
+        name = "SLACK_SIGNING_SECRET"
+        value_source {
+          secret_key_ref {
+            secret  = google_secret_manager_secret.slack_signing_secret.secret_id
+            version = "latest"
+          }
+        }
+      }
+
+      env {
+        name = "SLACK_BOT_USER_OAUTH_TOKEN"
+        value_source {
+          secret_key_ref {
+            secret  = google_secret_manager_secret.slack_bot_user_oauth_token.secret_id
+            version = "latest"
+          }
+        }
+      }
+
+      env {
+        name = "POTAL_TOKEN"
+        value_source {
+          secret_key_ref {
+            secret  = google_secret_manager_secret.potal_token.secret_id
+            version = "latest"
+          }
+        }
+      }
+
+      env {
+        name = "SECURITY_SALT"
+        value_source {
+          secret_key_ref {
+            secret  = google_secret_manager_secret.security_salt.secret_id
+            version = "latest"
+          }
+        }
+      }
+
+      env {
+        name = "SENTRY_FRONTEND_DSN"
+        value_source {
+          secret_key_ref {
+            secret  = google_secret_manager_secret.frontend_dsn.secret_id
+            version = "latest"
+          }
+        }
+      }
+
+      env {
+        name = "SENTRY_BACKEND_DSN"
+        value_source {
+          secret_key_ref {
+            secret  = google_secret_manager_secret.backend_dsn.secret_id
+            version = "latest"
+          }
+        }
+      }
+
+      env {
+        name = "DATABASE_USER"
+        value_source {
+          secret_key_ref {
+            secret  = google_secret_manager_secret.database_user.secret_id
+            version = "latest"
+          }
+        }
+      }
+
+      env {
+        name = "DATABASE_PASSWORD"
+        value_source {
+          secret_key_ref {
+            secret  = google_secret_manager_secret.database_password.secret_id
+            version = "latest"
+          }
+        }
+      }
+
+      env {
+        name = "DATABASE_NAME"
+        value_source {
+          secret_key_ref {
+            secret  = google_secret_manager_secret.database_name.secret_id
+            version = "latest"
+          }
+        }
+      }
+
+      env {
+        name = "DATABASE_SOCKET"
+        value_source {
+          secret_key_ref {
+            secret  = google_secret_manager_secret.database_socket.secret_id
+            version = "latest"
+          }
+        }
+      }
+    }
+  }
+
+  traffic {
+    type    = "TRAFFIC_TARGET_ALLOCATION_TYPE_LATEST"
+    percent = 100
+  }
+
+  depends_on = [
+    data.google_secret_manager_secret_version.security_salt,
+    data.google_secret_manager_secret_version.slack_client_secret,
+    data.google_secret_manager_secret_version.slack_signing_secret,
+    data.google_secret_manager_secret_version.slack_bot_user_oauth_token,
+    data.google_secret_manager_secret_version.potal_token,
+    google_sql_database_instance.db,
+  ]
```

Comment:
I don't think you need this, the reference to the resources themselves should set up the proper dependency tree for terraform.